samples/auth-user-sample/package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
In most of our other sample directories, we tend to have a single `samples/package.json` that holds the dependencies you need for all the samples.  I think having this bonus `package.json` is what's causing some of the CI weirdness.  Thinking maybe we should:
- delete this file
- move any dependencies you need to `samples/package.json`

Guessing that will help clean a fair bit of this up. @bcoe has the final call though :)

Reply:
I was told that a sample app should have its own package.json, but I don't know if there's actually a standard for this!

Reply:
If the goal is to demonstrate to the user a fully self contained application, I don't think it's unreasonable to have the inner package.json (this reminds me of how we've approached samples in our old samples mono repo):

https://github.com/GoogleCloudPlatform/nodejs-docs-samples/tree/master/appengine/cloudsql

If the goal is that we'd be writing upstream tutorials that embed this package.json, I think it's okay to have it in isolation. If this is not a requirement, it might be worth collapsing the dependencies into the `samples/package.json`.

Reply:
I think the most important thing is to show the install steps. Probably doesn’t need to embed the package.json if we add a “clone the repo” step to the tutorial.

Reply:
Alrighty, sounds like it's a good idea to just move dependencies to samples/package.json.